.circleci/config.yml

```diff
@@ -56,6 +56,7 @@ jobs:
       - run: CI=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
       - run: brew update && brew install nasm go wget qemu ent
       - run: brew tap nanovms/homebrew-x86_64-elf
```

Comment:
We added this tap before x86_64-elf-binutils was added to homebrew-core. Since it was added, the install step below has been silently preferring the one in homebrew-core, which seems to be just fine. So I would just remove these instances of 'brew tap'.

Reply:
OK, removed the nanovms/homebrew-x86_64-elf tap.